docs/UmlFiles/loadFile_sequence-diagram.puml

```diff
@@ -0,0 +1,113 @@
+@startuml
+title Notes File Parsing
+
+activate System
+
+--> NotesFileParser: loadFile(path)
+activate NotesFileParser
+NotesFileParser -> ArrayList: ArrayList()
+activate ArrayList
+ArrayList --> NotesFileParser
+deactivate ArrayList
+NotesFileParser -> Scanner: new Scanner(path)
+activate Scanner
+NotesFileParser -> Scanner: scanner.hasNextLine()
+loop for each line
+    Scanner -> Scanner: scanner.nextLine()
+    NotesFileParser -> NoteParser: parseNoteString(noteScanner)
+    activate NoteParser
+    NoteParser -> String: split()
+    activate String
+    String --> NoteParser
+    deactivate String
+    alt noteInfo.length == 4
+        alt priorityStr is valid
+            NoteParser -> NotePriority: Priority.valueOf(priorityStr)
+            activate NotePriority
+            NotePriority --> NoteParser
+            deactivate NotePriority
+        else priorityStr is invalid
+            NoteParser -> InvalidFormatException: throw exception
```

Comment:
Can add the activation bar everytime InvalidFormatException is called

![image](https://user-images.githubusercontent.com/88263677/230771550-2aaa0e76-a2f3-4f58-a879-819ed2d72478.png)

Reply:
I have added activation bar for InvalidFormatException when is called.